spec_helper.rb

```diff
@@ -48,10 +48,7 @@
   end
 end
 
-dir = "../fixtures/code"
-CODE_LOADING_DIR = defined?(require_relative) ?
-                     File.realpath(dir, __FILE__) :
-                     File.expand_path(dir, __FILE__)
+CODE_LOADING_DIR = File.expand_path "../fixtures/code", __FILE__
```

Comment:
It seems to be that since we have `require_relative`, features in $LOADED_FEATURES are expanded with realpath (their symlinks are resolved) while they used to not be before `require_relative`.
This in turn probably causes spec failures if executing rubyspec and the current directory contains symlinks.
@nobu Could you confirm I understand your commit correctly?